payments/controllers.go

```diff
@@ -0,0 +1,51 @@
+package payments
+
```

Comment:
What about using handlers rather than controllers as the naming convention for the file

Reply:
already have this convention elsewhere in the codebase...

Reply:
will be handled: https://github.com/brave-intl/bat-go/issues/1220